core/kernel/persistence/smoothsql/install/SmoothRdsModel.php

```diff
@@ -36,11 +36,11 @@ class SmoothRdsModel
      */
     public static function addSmoothTables(Schema $schema)
     {
-        $table = $schema->createTable("models");
-        $table->addColumn('modelid', "integer", ["notnull" => true,"autoincrement" => true]);
-        $table->addColumn('modeluri', "string", ["length" => 255,"default" => null]);
-        $table->addOption('engine', 'MyISAM');
-        $table->setPrimaryKey(['modelid']);
+//        $table = $schema->createTable("models");
+//        $table->addColumn('modelid', "integer", ["notnull" => true,"autoincrement" => true]);
+//        $table->addColumn('modeluri', "string", ["length" => 255,"default" => null]);
+//        $table->addOption('engine', 'MyISAM');
+//        $table->setPrimaryKey(['modelid']);
```

Comment:
Please remove the commented out code.

Reply:
yes, will do, it is for highlight the removal

Reply:
if the table is no longer created, should we not drop it during updates as well?

Reply:
I have some doubt about deleting table during update, do u think we can remove it?